nodes/viz/viewer_draw_mk4.py

```diff
@@ -185,17 +185,20 @@ def view_3d_geom(context, args):
         bgl.glLineWidth(1)
 
     if config.draw_verts:
-        bgl.glPointSize(config.point_size)
-        if config.uniform_verts:
-            v_batch = batch_for_shader(config.v_shader, 'POINTS', {"pos": geom.v_vertices})
-            config.v_shader.bind()
-            config.v_shader.uniform_float("color", config.vector_color[0][0])
-        else:
-            v_batch = batch_for_shader(config.v_shader, 'POINTS', {"pos": geom.v_vertices, "color": geom.points_color})
-            config.v_shader.bind()
+        if len(geom.v_vertices)>0 and (len(geom.v_vertices[0])==3) == True:
```

Comment:
`(len() == 3) == True` looks odd. Why not just `len() == 3`?

Reply:
To emphasize count of vertices. )
Should I skip this and remove “== True” ?

Reply:
https://pep8.org/#programming-recommendations

![image](https://github.com/nortikin/sverchok/assets/28003269/133ebcad-62b2-4ea3-b50a-90de572507ac)

Reply:
@Durman  Is it good now? )